src/Hl7.Fhir.ElementModel/ICdaInfoSupplier.cs

```diff
@@ -8,8 +8,11 @@
 
 namespace Hl7.Fhir.ElementModel
 {
-    public interface ICcdaInfoSupplier
+    public interface ICdaInfoSupplier
```

Comment:
Or Hl7v3InfoSupplier?  I don't even think this is CDA specific?  Well, the use of XHTML may be, but there might be other information (like information about the original xsi:type attribute in the source data) that you make available here, and this is more general for the v3 serialization, and not specific to CDA.

Reply:
It's not CDA specific - we want more general "give me the original wire-representation of this node" functionality here.  This is connected to how SourceNode and TypedElement cooperate on nodes that should actually not be parsed.  SourceNode has no type information present to deal with that, but TypedElement is too late to recombine these nodes into their original representation. See also .NET API issue 1274.